tests/ApiPortVS.Tests/OptionsViewModelTests.cs

```diff
@@ -1,98 +0,0 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using ApiPortVS.Contracts;
-using ApiPortVS.Options;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using NSubstitute;
-
-namespace ApiPortVS.Tests
```

Comment:
Why the removal of these unit tests?

Reply:
It was a duplicated test (must have been renamed at some point to `OptionsViewModelTest` but this duplicated existed. It also still uses MSTest while the other one uses Xunit